test/Libraries/DynamoMSOfficeTests/ExcelTests.cs

```diff
@@ -9,1362 +9,11 @@
 namespace Dynamo.Tests
 {
 
-#if NET6_0_OR_GREATER
     [TestFixture, Apartment(ApartmentState.MTA)]
     public class ExcelTests
     {
     }
```

Comment:
Can you remind me why we didn't have any excel tests for net6?

Reply:
https://github.com/DynamoDS/Dynamo/pull/14041#discussion_r1212472854
Looks like we simply forgot to do it after all projects were migrated
I will enable them